src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -94,4 +120,60 @@ impl Kbc {
             .ok_or_else(|| anyhow!("TEE rsa key missing"))?;
         response.decrypt_output(key)
     }
+
+    fn tee(&self) -> String {
+        self.tee.clone()
+    }
+
+    fn kbs_uri(&self) -> String {
+        self.kbs_uri.clone()
+    }
+
+    fn kbs_client(&mut self) -> Result<&mut reqwest::Client> {
+        if let Some(kbs_client) = &mut self.kbs_client {
+            Ok(kbs_client)
+        } else {
+            Err(anyhow!("Start kbs http client failed"))
+        }
+    }
+
+    async fn request_kbs_resource(&mut self, resource_url: String) -> Result<Response> {
+        let mut res = self.kbs_client()?.get(&resource_url).send().await?;
```

Comment:
(optimization)
The first time this is called it will surely fail, because the kbs_client has no cookies.
Maybe we can locally check if the cookie store is empty, and if it is, start directly from the `POST /auth` request.

Reply:
This makes sense, but the cookie store in reqwest client is private, which is inconvenient. I use explicit cookie store to achieve this.